test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -32,6 +32,8 @@ def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
 @pytest.mark.parametrize("ec2_instance_type", SMDEBUG_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_smdebug_cpu(training, ec2_connection, region, cpu_only, py3_only):
     # TODO: Remove this once test timeout has been debugged (failures especially on m4.16xlarge)
+    if is_tf2(training) and "2.3.0" in training and "m4.16xlarge" in SMDEBUG_EC2_CPU_INSTANCE_TYPE:
+        pytest.skip("Currently skipping for TF2.3.0 on m4.16xlarge until the issue is fixed")
```

Comment:
nit: Could you put a skipif on the function which checks the value of `os.getenv("EC2_CPU_INSTANCE_TYPE")` so that the lead-up time to create an instance for this test is also skipped?